Code/Editor/Plugins/ComponentEntityEditorPlugin/UI/Outliner/OutlinerListModel.cpp

```diff
@@ -2605,8 +2605,6 @@ void OutlinerItemDelegate::paint(QPainter* painter, const QStyleOptionViewItem&
         AzToolsFramework::RichTextHighlighter::PaintHighlightedRichText(
             entityNameRichText, painter, optionV4, textRect, QPoint(0, verticalOffset));
 
-        painter->restore();
```

Comment:
Looking at the top of this `index.column() == OutlinerListModel::ColumnName` branch it actually does a `painter->save()` on line 2549, but then never restores it, so I think we actually should add one here